lib/libsundials_api.jl

```diff
@@ -1,14 +1,19 @@
-function ARKStepCreate(fe::ARKRhsFn, fi::ARKRhsFn, t0::realtype, y0::N_Vector)
+# Modified from autogenerated code to fix N_Vector https://github.com/SciML/Sundials.jl/issues/367
+# Global edits:
+#    convert(N_Vector, x) -> identity(x)
```

Comment:
why `identity(x)` instead of `x`?

Reply:
because that could easily be done with a global edit (there's ~400 of them...)

Reply:
now updated to fix this